projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.spec.ts

```diff
@@ -281,17 +281,21 @@ describe('PropertyViewComponent', () => {
 
             let addButtons = propertyViewComponentDe.queryAll(By.css('button.create'));
 
-            // current amount of buttons should equal 19 because the boolean property shouldn't have an add button if it has a value
-            expect(addButtons.length).toEqual(19);
+            // current amount of buttons should equal 17
+            // because the boolean property shouldn't have an add button if it has a value
+            // standoff links value and has incoming link value are system props and cannot be added: -2
+            expect(addButtons.length).toEqual(17);
 
             // remove value from the boolean property
             testHostComponent.propArray[9].values = [];
 
             testHostFixture.detectChanges();
 
-            // now the boolean property should have an add button so the amount of add buttons on the page should increase by 1
+            // now the boolean property should have an add button
+            // so the amount of add buttons on the page should increase by 1
+            // standoff links value and has incoming link value are system props and cannot be added: -2
             addButtons = propertyViewComponentDe.queryAll(By.css('button.create'));
-            expect(addButtons.length).toEqual(20);
+            expect(addButtons.length).toEqual(18);
```

Comment:
Is there a better way to know the number of expected buttons than to write a hardcoded number? Maybe we can use the propArray to check how many properties are not read-only and use that? This way if the test data changes, we don't have to keep manually updating the number. Idk how it will work with cardinalities though.

Reply:
I added some explicit tests in https://github.com/dasch-swiss/dsp-ui-lib/pull/233/commits/186d0246daea55f5c9b547de4eac5aa5a1346062